src/PROPOSAL/PROPOSAL/secondaries/parametrization/photonuclear/Photonuclear.h

```diff
@@ -31,12 +31,12 @@ namespace secondaries {
             primary_lepton.propagated_distance = 0.;
 
             auto hadron = ParticleState();
```

Comment:
This does an unnecessary copy, correct?
```
ParticleState hadron;
```

Reply:
Yes, you're right. Still, I think `ParticleState hadron;` is the canonical form, shorter and thus more readable.